### DISCUSSION TITLE
Validation of unique constraint using form object

### DISCUSSION THREAD

**Discussion by jumawilliam** [livewire/livewire#6691] [1. Help]
### Livewire version

v3.o

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

Instead of checking for email column its checking for form.email.  

I get the error: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'form.email' in 'where clause'

My form  object 
<?php

namespace App\Livewire\Forms;

use Livewire\Attributes\Rule;
use Livewire\Form;

class RegisterForm extends Form
{
    #[Rule('required', message: 'Please enter your name')]
    public $name;

    #[Rule('required|email|unique:users')]
    public $email;

    #[Rule('required|min:8')]
    public $password;
}

My livewire Class
<?php

namespace App\Livewire;

use App\Livewire\Forms\RegisterForm;
//use Livewire\Attributes\Rule;
use App\Models\User;
use Livewire\Component;

class RegisterUser extends Component
{
    public RegisterForm $form;

    public function render()
    {
        return view('livewire.register-user');
    }

    public function save()
    {
        $this->form->validate();
       

    }
}

The livewire view 

<div class="offset-3 col-6">
    <form wire:submit="save">
        <div class="mb-3">
            <label class="form-label">Name</label>
            <input wire:model="form.name" type="text" class="form-control" >
            <div>
                @error('form.name') <span class="text-danger">{{ $message }}</span> @enderror 
            </div>
         
        </div>
        <div class="mb-3">
          <label class="form-label">Email address</label>
          <input wire:model="form.email" type="text" class="form-control" >
          <div>
            @error('form.email') <span class="text-danger">{{ $message }}</span> @enderror 
        </div>    
        </div>
        <div class="mb-3">
          <label class="form-label">Password</label>
          <input wire:model="form.password" type="password" class="form-control" >
          <div>
            @error('form.password') <span class="text-danger">{{ $message }}</span> @enderror 
        </div>    
        </div>
        <div class="mb-3 form-check">
          <input wire:mode="remember" type="checkbox" class="form-check-input">
          <label class="form-check-label" for="exampleCheck1">Remember</label>
        </div>
        <button type="submit" class="btn btn-primary">
            Submit
            <div wire:loading class="spinner-border spinner-border-sm" role="status">
                
            </div>
        </button>
      </form>
</div>

### Additional information

The error is caused by this line in the RegisterForm 

 #[Rule('required|email|unique:users')]
    public $email;

Where we are checking against our database that we dont have a similar email

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@jumawilliam you should just specify the column after the table name, like `unique:users,email`. https://laravel.com/docs/10.x/validation#rule-unique

Hope this helps!
